cpp/src/parquet/arrow/reader_writer_benchmark.cc

```diff
@@ -197,6 +197,52 @@ BENCHMARK_TEMPLATE2(BM_WriteColumn, true, DoubleType);
 BENCHMARK_TEMPLATE2(BM_WriteColumn, false, BooleanType);
 BENCHMARK_TEMPLATE2(BM_WriteColumn, true, BooleanType);
 
+int32_t kInfiniteUniqueValues = -1;
+
+std::shared_ptr<::arrow::Table> RandomStringTable(int64_t length, int64_t unique_values,
+                                                  int64_t null_percentage) {
+  std::shared_ptr<::arrow::DataType> type = ::arrow::utf8();
+  std::shared_ptr<::arrow::Array> arr;
+  ::arrow::random::RandomArrayGenerator generator(500);
```

Comment:
Is this the seed?

Reply:
Yeah, the same seed is hardcoded elsewhere in the file. Is there somewhere we should be taking the seed from?

Reply:
No, it's ok. I would just suggest naming the parameter, e.g.:
```suggestion
  ::arrow::random::RandomArrayGenerator generator(/*seed=*/ 500);
```